src/ui/input/captcha_input.jsx

```diff
@@ -54,7 +54,7 @@ export default class CaptchaInput extends React.Component {
         </div>
         <InputWrap
           focused={focused}
-          invalidHint={invalidHint}
+          invalidHint=""
```

Comment:
Is this intended?

Reply:
No, this is actually intended.. The component is generic and it requires an invalidHint parameter, the `invalidHint` text is shared for all components.

Tony suggested this text get replaced here and I didn't know what to use

https://auth0.slack.com/archives/GG1ENH909/p1594070548262000?thread_ts=1594069365.258500&cid=GG1ENH909

I think it doesn't look bad for the auth0 case, since the description is above like in the header of the widget